src/System.Security.Cryptography.Encoding/ref/System.Security.Cryptography.Encoding.cs

```diff
@@ -20,6 +20,49 @@ public partial class AsnEncodedData
         public virtual void CopyFrom(System.Security.Cryptography.AsnEncodedData asnEncodedData) { }
         public virtual string Format(bool multiLine) { return default(string); }
     }
+    public sealed partial class AsnEncodedDataCollection : System.Collections.ICollection, System.Collections.IEnumerable
+    {
+        public AsnEncodedDataCollection() { }
+        public AsnEncodedDataCollection(System.Security.Cryptography.AsnEncodedData asnEncodedData) { }
+        public int Count { get { return default(int); } }
+        public System.Security.Cryptography.AsnEncodedData this[int index] { get { return default(System.Security.Cryptography.AsnEncodedData); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
```

Comment:
It gets extracted to Tools\GenAPI.exe, which is the buildtools latest version.  It may have gotten repacked into a different nuget package.